.github/workflows/ci-windows.yml

```diff
@@ -0,0 +1,27 @@
+name: Windows CI
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
why not just run on fedora latest directly?

Reply:
GitHub only provides Ubuntu as a virtual environment for Linux:
https://github.com/actions/virtual-environments

Reply:
Oh right, I forgot about that. Doh.